composer.json

```diff
@@ -2,18 +2,18 @@
   "name": "magento/composer",
   "description": "Magento composer library helps to instantiate Composer application and run composer commands.",
   "type": "library",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "license": [
     "OSL-3.0",
     "AFL-3.0"
   ],
   "require": {
-    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0",
-    "composer/composer": "1.4.1",
-    "symfony/console": "~2.3, !=2.7.0"
+    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0",
```

Comment:
As `symfony/console` supports PHP `~7.1.3`, we should remove older versions from the list of supported here.
Also, need confirmation from @piotrekkaminski that it's ok to support PHP starting from 7.1.3. If not, we should upgrade to symfony 3 only.